streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -1907,13 +1937,38 @@ public TimestampExtractor defaultTimestampExtractor() {
         return getConfiguredInstance(DEFAULT_TIMESTAMP_EXTRACTOR_CLASS_CONFIG, TimestampExtractor.class);
     }
 
+    public DeserializationExceptionHandler deserializationExceptionHandler() {
+        if (getClass(DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG) != null &&
```

Comment:
`getClass` would return the default if not set. I think we would need to use
```
originals().containsKey(DESERIALIZATION_EXCEPTION_HANDLER_CLASS_CONFIG)
```

we should add a test if the log line is printed, too (cf helper `LogCaptureAppender.java`)

Reply:
thanks for the tip.